src/api-reference/request/examples.markdown

```diff
@@ -0,0 +1,755 @@
+
+
+<a name="expense-resource"></a>
+### expense-resource
+<i>Manage expected expense entries attached to a Request document</i><br />
+
+#### DELETE /v4/expenses/{expenseUuid}
+<code>curl -X DELETE --header 'Accept: application/json' --header 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IjE0NTU2MTQzNDYifQ.eyJjb25jdXIuc2NvcGVzIjpbInRyYXZlbHJlcXVlc3Qud3JpdGUiXSwiYXVkIjoiKiIsImNvbmN1ci5wcm9maWxlIjoiaHR0cHM6Ly9ycWEzLmFwaS5jb25jdXJhc3AuY29tL3Byb2ZpbGUvdjEvcHJpbmNpcGFscy84OWRkN2M5MC02ZWY0LTQxZDEtOWFlOC0zMDNiYWQyNTg3NGQiLCJjb25jdXIudmVyc2lvbiI6MywiY29uY3VyLnR5cGUiOiJ1c2VyIiwiY29uY3VyLmFwcCI6Imh0dHBzOi8vcnFhMy5hcGkuY29uY3VyYXNwLmNvbS9wcm9maWxlL3YxL2FwcHMvNzhjMzg4MWMtM2I2NS00YjgxLWE0YzAtNjI0ZDQxZDBjZmNlIiwic3ViIjoiODlkZDdjOTAtNmVmNC00MWQxLTlhZTgtMzAzYmFkMjU4NzRkIiwiaXNzIjoiaHR0cHM6Ly9ycWEzLmFwaS5jb25jdXJhc3AuY29tIiwiZXhwIjoxNTIyODU1NjM3LCJjb25jdXIuYXBwSWQiOiI3OGMzODgxYy0zYjY1LTRiODEtYTRjMC02MjRkNDFkMGNmY2UiLCJuYmYiOjE1MjI4NTIwMzcsImlhdCI6MTUyMjg1MjAzNywiY29uY3VyLmNvbXBhbnkiOiIwNWIzYjc4Zi04OTMwLTRmMWUtOTY4MS03NDMxYzkzMmM0MDIifQ.P1OHS4QYgFMSCGIojLox3bxezRyg1MYWwwwbZ7Q9EWYx9XhKNrR7QUyOMyn1o-iW78SliSYrIFtpXnWOKbLYrhkZojlgyEIEM8ENX3suSii876W9DAmvJtM9q98UglUCN4mBa96WjE1-VXWghP4yW0WG9tLc5IZg0WLCOIxBrUY4qmAVu4dUijAvWpkg-YUiP0nn2o4rx-cfWZvxql0OIQyPMLtue7Tyb96pER8u5MJfgpWSnuZOsDUU3youJa4RzB8xILAO4Q9IFVFfPrq43S80oo4YwmSquFVTTTngZL82d08d9HhjheXu2t_Cude-YOUABNYeAb7yJVyu9JMx8g' 'https://rqa3-raas.concurasp.com/v4/expenses/B4D37C35E16E904F90C182D5751D3440'</code>
+```json
+true
+```
+Response code : **200**
+
+#### GET /v4/expenses/{expenseUuid}
+<code>curl -X GET --header 'Accept: application/json' --header 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IjE0NTU2MTQzNDYifQ.eyJjb25jdXIuc2NvcGVzIjpbInRyYXZlbHJlcXVlc3Qud3JpdGUiXSwiYXVkIjoiKiIsImNvbmN1ci5wcm9maWxlIjoiaHR0cHM6Ly9ycWEzLmFwaS5jb25jdXJhc3AuY29tL3Byb2ZpbGUvdjEvcHJpbmNpcGFscy84OWRkN2M5MC02ZWY0LTQxZDEtOWFlOC0zMDNiYWQyNTg3NGQiLCJjb25jdXIudmVyc2lvbiI6MywiY29uY3VyLnR5cGUiOiJ1c2VyIiwiY29uY3VyLmFwcCI6Imh0dHBzOi8vcnFhMy5hcGkuY29uY3VyYXNwLmNvbS9wcm9maWxlL3YxL2FwcHMvNzhjMzg4MWMtM2I2NS00YjgxLWE0YzAtNjI0ZDQxZDBjZmNlIiwic3ViIjoiODlkZDdjOTAtNmVmNC00MWQxLTlhZTgtMzAzYmFkMjU4NzRkIiwiaXNzIjoiaHR0cHM6Ly9ycWEzLmFwaS5jb25jdXJhc3AuY29tIiwiZXhwIjoxNTIyODUwMjMzLCJjb25jdXIuYXBwSWQiOiI3OGMzODgxYy0zYjY1LTRiODEtYTRjMC02MjRkNDFkMGNmY2UiLCJuYmYiOjE1MjI4NDY2MzMsImlhdCI6MTUyMjg0NjYzMywiY29uY3VyLmNvbXBhbnkiOiIwNWIzYjc4Zi04OTMwLTRmMWUtOTY4MS03NDMxYzkzMmM0MDIifQ.S9trem03nqas3tJmWeI13q7Dcim_IWeWTW-S8YiQ7TkErtU1CLOgwr-XckVTz4trkteiz5XGMFc1pkNjTCGyHRYSVyWEs7GzcEoygo6Ub7cQo3S1Sk9tr0aW3eV7rHQFdUVobGs1OIbQWiHfUjrl8jEQm8aZOEMJcACJUPMPIbmxrlKr8gkNnEmLZkcBZuC96iYLkRh7y5_8NTpWe0uM8j18MCPnBwG5Kjka1aaD0tg0UAu9bFnf74EEa8OE9dRa2Y5AXZmNHQ5n0fvvoWoREL2lGLUtCFYu9NgxZpd5hHGxj4EyPzbdSg4bXEapZCgK87xclj2n1jydyXaHmxjkLA' 'https://rqa3-raas.concurasp.com/v4/expenses/B4D37C35E16E904F90C182D5751D3440'</code>
+```json
+{
+  "href": "https://rqa3-raas.concurasp.com/v4/expenses/B4D37C35E16E904F90C182D5751D3440",
```

Comment:
Should we use production hostnames or a hostname placeholder instead of RQA3 hostname ?